From b6bc5e4d11218e19d00b6d5f713a1706adf6b02f Mon Sep 17 00:00:00 2001
From: Rajesh Bhagat <rajesh.bhagat@nxp.com>
Date: Tue, 16 Jan 2018 15:07:52 +0530
Subject: [PATCH 04/31] ls1043a: dcm: Change serdes for supporting SGMII

Changes serdes for supporting SGMII.

NOTE: currently using 0x333, will be changed later.

Signed-off-by: Prabhakar Kushwaha <prabhakar.kushwaha@nxp.com>
Signed-off-by: Rajesh Bhagat <rajesh.bhagat@nxp.com>
---
 board/freescale/ls1043ardb/ls1043adcm_rcw_nand.cfg | 2 +-
 board/freescale/ls1043ardb/ls1043adcm_rcw_sd.cfg   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/board/freescale/ls1043ardb/ls1043adcm_rcw_nand.cfg b/board/freescale/ls1043ardb/ls1043adcm_rcw_nand.cfg
index 4e3a5fa946..b325069215 100644
--- a/board/freescale/ls1043ardb/ls1043adcm_rcw_nand.cfg
+++ b/board/freescale/ls1043ardb/ls1043adcm_rcw_nand.cfg
@@ -2,6 +2,6 @@
 aa55aa55 01ee0100
 # serdes protocol
 08100010 0a000000 00000000 00000000
-14550002 80004012 e0106000 c1002000
+33330002 00404012 e0106000 c1002000
 00000000 00000000 00000000 00038800
 20124000 00061100 00000096 00000001
diff --git a/board/freescale/ls1043ardb/ls1043adcm_rcw_sd.cfg b/board/freescale/ls1043ardb/ls1043adcm_rcw_sd.cfg
index 4022f1ee3b..1479e1d9c3 100644
--- a/board/freescale/ls1043ardb/ls1043adcm_rcw_sd.cfg
+++ b/board/freescale/ls1043ardb/ls1043adcm_rcw_sd.cfg
@@ -2,6 +2,6 @@
 aa55aa55 01ee0100
 # RCW
 08100010 0a000000 00000000 00000000
-14550002 80004012 60040000 c1002000
+33330002 00404012 60040000 c1002000
 00000000 00000000 00000000 00038800
 20124000 00061100 00000096 00000001
-- 
2.17.1

